Clean all the buttons

capilkey opened the pull request:
This PR was just supposed to fix the wiggle in the mute/unmute button, but I had to dig deep to find the root cause of the sizing issues. I ended up simplifying a lot of the button sizing for the circle buttons specifically.

I also restyled the chat header after the polling header, made the join audio action button a headphone icon when in listen-only, and improved the focus indicator for Firefox and Windows high contrast mode.

Closes #6934

vitormateusalmeida commented:
In Chrome, buttons has a square border on focus.
![image](https://user-images.githubusercontent.com/9675166/54847995-bc68e900-4cbe-11e9-86b6-75a6b9bb15bf.png)  ![image](https://user-images.githubusercontent.com/9675166/54848036-e4584c80-4cbe-11e9-86e9-8a67d4ff14a3.png)

Add `outline:0;` in button style to solve this

capilkey commented:
Fixed the outlines. I removed all of the `outline: 0` or `outline: none` because they break accessibility for high contrast modes. The recommendation was to use a transparent outline and that's what I had done, but I added it to the span instead of setting it on the outer <button>
